sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -100,7 +100,7 @@ platform. First, add the package:
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.0.3</version>
+    <version>1.1.2</version>
```

Comment:
Unrelated to this PR, but should we remove this from the readme. I don't think thats the pattern we follow for other SDK's readme to have identity version info in the readme.

Reply:
I see that all services have this in readme. It's required in the "Authenticate the client" section.

Reply:
Oh yes! I was confused if we actually kept it or not, you are good!